neutron/assetlist.json

```diff
@@ -747,6 +747,46 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/retro.svg"
         }
       ]
+    },
+    {
+      "description": "The first memecoin on osmosis.",
+      "denom_units": [
+        {
+          "denom": "ibc/7DA39F5140741177846FCF3CFAB14450EE7F57B7794E5A94BEF73825D3741958",
+          "exponent": 0,
```

Comment:
This comma without a property following will break the schema

Reply:
Done (added archway too)